framework/ui/src/components/organisms/vc-dynamic-property/vc-dynamic-property.vue

```diff
@@ -2,7 +2,8 @@
   <vc-select
     v-if="property.dictionary || property.isDictionary"
     :label="
-      (property.displayNames && property.displayNames[0].name) || property.name
+      (property.displayNames && property.displayNames[0].name) ||
```

Comment:
Here we need to get proper locale instead of first from array. Need to improve later.